presto-iceberg/src/test/java/com/facebook/presto/iceberg/TestIcebergSystemTables.java

```diff
@@ -203,6 +203,25 @@ public void testFilesTable()
         assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$files\"");
     }
 
+    @Test
+    public void testRefsTable()
+    {
+        assertQuery("SHOW COLUMNS FROM test_schema.\"test_table$refs\"",
+                "VALUES ('name', 'varchar', '', '')," +
+                        "('type', 'varchar', '', '')," +
+                        "('snapshot_id', 'bigint', '', '')," +
+                        "('max_reference_age_in_ms', 'bigint', '', '')," +
+                        "('min_snapshots_to_keep', 'bigint', '', '')," +
+                        "('max_snapshot_age_in_ms', 'bigint', '', '')");
+        assertQuerySucceeds("SELECT * FROM test_schema.\"test_table$refs\"");
+
+        // Check main branch entry
+        assertQuery("SELECT count(*) FROM test_schema.\"test_table$refs\"", "VALUES 1");
+        assertQuery("SELECT name FROM test_schema.\"test_table$refs\"", "VALUES 'main'");
```

Comment:
I think it would also be prudent to add assertions on values in the row of the table matching the `SnapshotRef` object.

You could add or refactor the `loadTable` method from `IcebergDistributedTestBase` to get a reference to the Iceberg table, then re-create the values and run `assertQuery`

Reply:
Added tests in`IcebergDistributedTestBase` which I could reuse in my subsequent PR around adding support for querying tags & branch. Please review